package.json

```diff
@@ -50,60 +56,31 @@
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
+    "@redhat-cloud-services/frontend-components-config": "0.0.6",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^24.1.0",
-    "babel-loader": "^8.0.4",
     "babel-plugin-dual-import": "^1.2.1",
     "babel-plugin-lodash": "^3.3.4",
-    "clean-webpack-plugin": "^1.0.0",
-    "connect-history-api-fallback": "^1.5.0",
-    "copy-webpack-plugin": "^4.6.0",
-    "css-loader": "^1.0.1",
-    "d3": "^5.7.0",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.7.0",
     "enzyme-to-json": "^3.3.4",
     "eslint": "^5.13.0",
     "eslint-config-prettier": "^3.3.0",
-    "eslint-loader": "^2.1.1",
     "eslint-plugin-react": "^7.11.1",
     "fetch-mock": "^7.3.1",
-    "file-loader": "^2.0.0",
-    "git-revision-webpack-plugin": "^3.0.3",
-    "glob": "^7.1.3",
-    "html-replace-webpack-plugin": "^2.5.3",
-    "html-webpack-plugin": "^3.2.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.7.1",
     "jest-cli": "^24.7.1",
-    "koa-connect": "^2.0.1",
-    "lodash-webpack-plugin": "^0.11.5",
-    "mini-css-extract-plugin": "^0.4.4",
-    "node-sass": "^4.12.0",
+    "node-fetch": "^2.6.0",
     "npm-run-all": "^4.1.3",
-    "react": "^16.6.1",
-    "react-bootstrap": "^0.32.4",
-    "react-dom": "^16.6.1",
-    "react-redux": "^5.1.1",
-    "react-router-dom": "^4.3.1",
-    "redux": "^4.0.1",
-    "redux-logger": "^3.0.6",
-    "redux-promise-middleware": "^5.1.1",
-    "sass-loader": "^7.1.0",
-    "source-map-loader": "^0.2.4",
-    "style-loader": "^0.23.1",
     "stylelint": "^9.10.1",
     "stylelint-config-recommended-scss": "^3.2.0",
     "stylelint-scss": "^3.4.0",
     "webpack": "^4.25.1",
     "webpack-bundle-analyzer": "^3.3.2",
     "webpack-cli": "^3.1.2",
-    "webpack-hot-client": "^4.1.1",
-    "webpack-serve": "1.0.2",
-    "webpack-visualizer-plugin": "^0.1.11",
-    "whatwg-fetch": "^3.0.0",
-    "write-file-webpack-plugin": "^4.4.1"
```

Comment:
i believe we need the "webpack-dev-server": "3.4.1" dep here

Reply:
This dependency comes from https://github.com/RedHatInsights/frontend-components/blob/master/packages/config/package.json#L37